Tests/SentryTests/Integrations/SessionReplay/SentrySessionReplayIntegrationTests.swift

```diff
@@ -68,6 +81,21 @@ class SentrySessionReplayIntegrationTests: XCTestCase {
         expect(SentrySDK.currentHub().trimmedInstalledIntegrationNames().count) == 1
         expect(SentryGlobalEventProcessor.shared().processors.count) == 1
     }
+    
+    func testWaitForNotificationWithNoWindow() {
+        uiApplication.windowsMock = nil
+        startSDK(sessionSampleRate: 1, errorSampleRate: 0)
+        
+        guard let sut = SentrySDK.currentHub().installedIntegrations().first as? SentrySessionReplayIntegration else {
+            fail("Did not installed replay integration")
```

Comment:
`m`: That looks like it could get brittle because what if the order of the installed integration changes? I think filtering for one integration to be the `SentrySessionReplayIntegration` would be a better approach.

Reply:
If we change the order of the integrations in the test (it should not happen) this will immediately fail and then we fix the integration index.

Reply:
Up to you. I'm also fine with the way it is.